src/main/java/com/jcabi/maven/plugin/AjcMojo.java

```diff
@@ -187,7 +198,35 @@ public void execute() throws MojoFailureException {
         if (this.classesDirectory.mkdirs()) {
             Logger.info(this, "Created classes dir %s", this.classesDirectory);
         }
-        if (this.hasClasses() || this.hasSourceroots()) {
+        final boolean projectHasClasses = this.hasClasses();
+        if (!this.unwovenClassesDir.equals(this.classesDirectory)) {
+            this.unwovenClassesDir.mkdirs();
+            Logger.info(
+                this,
+                "Unwoven classes will be copied to %s",
```

Comment:
@amihaiemil according to indentation rule, we should try putting as much as possible in each line, so e.g. these two should be joined